### PR TITLE
Switch graph header to be an <input />

### PR DESCRIPTION
**Pull request by girum11**
#99

### CODE REVIEW

File: src/client/app/shared/graph-header/graph-header.component.html [hunk 1]
```diff
@@ -7,4 +7,4 @@
 
 <!-- TODO(girum): This app header should be wired up to the Google Drive
                   file that's currently open, and be rename-able. -->
-<h1>Untitled App Flow</h1>
\ No newline at end of file
+<input class="graph-header-title" [(ngModel)]="graphName">
```
Comment (eyuelt) [inline src/client/app/shared/graph-header/graph-header.component.html]: Missing the "/" to close the tag.

Comment (girum11) [inline src/client/app/shared/graph-header/graph-header.component.html]: Ah good eye, fixed.

**Comment by eyuelt** [comment]
LGTM

**Comment by eyuelt** [comment]
LGTM

**Pull request merged**